### DISCUSSION TITLE
Is there a Retraction before Parked head returns to print after Filment Out?

### DISCUSSION THREAD

**Discussion by Ergonomicmike** [mriscoc/Ender3V2S1#801] [General]
I tried Filament Detection. It works.

But after purging, when I click "Continue," the head returns to its previous print location with a bit of filament oozing out from the last purge. (I pulled away the dried string before clicking "Continue.")

Is there a retraction before the head returns to print?

**Comment by mriscoc** [comment]
Wait a little to press continue to free the filament pressure in the hot end, clean and continue. There is a little retract but if the pressure is big that can't prevent the oozing.
